guides/common/assembly_building_cloud-images.adoc

```diff
@@ -14,8 +12,4 @@ include::modules/proc_installing-the-katello-agent.adoc[leveloffset=+1]
 
 include::modules/proc_installing-and-configuring-the-puppet-agent.adoc[leveloffset=+1]
 
-include::modules/proc_completing-the-rhel7-image.adoc[leveloffset=+1]
```

Comment:
Do we need to restore this as well? @ekohl

Reply:
All changes in this file need to be reverted.

Reply:
Addressed :+1: